src/main/java/com.launchcode.munchincrunch/models/FavoriteRestaurantEntity.java

```diff
@@ -0,0 +1,68 @@
+package com.launchcode.munchincrunch.models;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Table(name = "favorite_restaurants") //maps to MySQL
+public class FavoriteRestaurantEntity {
+    @Id //primary key for fav rest. table
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+
+    private Long id;
+    private String name;
+    private double rating;
+    private String address;
+    private String number;
+
+// Constructors, getters, setters
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public double getRating() {
+        return rating;
+    }
+
+    public void setRating(double rating) {
+        // Set rating only if it's greater than or equal to 4
+        if (rating >= 4) {
```

Comment:
Why rating much be higher ? where are we getting this rating from ?

Reply:
This was to make a commit and can remove it team doesn't like idea of pulling only in good ratings for dining spos.